templates/helm/values.yaml.tpl

```diff
@@ -85,3 +85,10 @@ serviceAccount:
   name: {{ .ServiceAccountName }}
   annotations: {}
     # eks.amazonaws.com/role-arn: arn:aws:iam::AWS_ACCOUNT_ID:role/IAM_ROLE_NAME
+
+# controller reconciliation configurations
+reconcile:
+  # The default duration, in seconds, to wait before resyncing desired state of custom resources.
+  defaultResyncPeriod: 0
+  # An object representing the reconcile resync configuration for.
+  resourceResyncPeriod: {}
```

Comment:
I think the proper grammar for this would be `resourceResyncPeriods` (note the plural). Multiple periods because there are multiple resources.